README.md

```diff
@@ -44,24 +44,45 @@ php-fpm with PHP 7.3
 
 Sphinx search service (service-sphinx.yml). 
 
+#### Installing Sphinx
+
 Before enabling make sure that:
 - Your database is named vanilla_dev
 - You have set `Plugins.Sphinx.Server = sphinx` in your config
 - You have set `Plugins.Sphinx.SphinxAPIDir = /sphinx/` in your config
 - You have enabled the sphinx plugin
-- You symlinked one of the configs-available as sphinx.conf in resources/usr/local/etc/sphinx/conf.d
+- You symlinked one of the [configs-available](./resources/usr/local/etc/sphinx/configs-available) as sphinx.conf in resources/usr/local/etc/sphinx/conf.d
 - Example from conf.d/: `ln -s configs-available/standard.sphinx.conf sphinx.conf`
-- If you need to have Sphinx indexes updated regularly run `./install-sphinx-cron.sh`
-  - by default it will reindex delta indexes every minute and reindex all indexes every 5 min.
-  - by default cron jobs hit `sphinx` container
-  - if your environment or tasked to be different you need to change  `install-sphinx-cron.sh`
+
+#### Re-indexing your database
+
+There are a couple of handly scripts to run the re-indexer. You can run them from the command line like so:
+
+```
+docker exec -t sphinx bash /root/index.delta.sh
+docker exec -t sphinx bash /root/index.all.sh
+```
+
+#### Installing the Sphinx indexr crontab
+
+If you need to have Sphinx indexes updated regularly run [install-sphinx-cron.sh](./images/sphinx/root/install-sphinx-cron.sh).
```

Comment:
I don't think this needs any installation.

The CRON is automatically installed when you build the container.

https://github.com/vanilla/vanilla-docker/blob/f9aa7fb6d251363f9fa2e47a39132fac897e45cb/images/sphinx/Dockerfile#L14

Reply:
It's definitely working as of this morning as well for Alexei and I.

Reply:
I don't know what to do here. I had just reformatted the existing docs and put a link to the script that was not easy for me to find.